.gitlab-ci.yml

```diff
@@ -78,3 +134,34 @@ deploy:deploy-rnode-artifacts:
     - testing
     - /^release-.*$/
     - tags 
+
+
+deploy:deploy-rnode-artifacts-java10:
+  stage: deploy
+  tags:
+    - preloaded-ubuntu1804-java10
+  script:
+    - ./scripts/install_bnfc.sh
+    # build rosette
+    - cd rosette
+    - ./build.sh
+    - cd ..
+    # build deb/rpm packages and compressed archive
+    - sudo sbt -Dsbt.log.noformat=true clean rholang/bnfc:generate casper/test:compile node/rpm:packageBin node/debian:packageBin node/universal:packageZipTarball 
+    - set -eo pipefail
+    - artifacts_dir=$(mktemp -d /tmp/artifacts_out.XXXXXXXX)
+    - cp node/target/rnode_*_all.deb ${artifacts_dir}/
+    - cp node/target/rpm/RPMS/noarch/rnode-*.noarch.rpm ${artifacts_dir}/
+    - cp rosette/build.out/rosette-*.deb ${artifacts_dir}/
+    - cp node/target/universal/*.tgz ${artifacts_dir}/
+    # deploy artifacts to dispatcher
+    - mkdir -p ~/.ssh
+    - echo "${SSH_PRIVATE_KEY}" | tr -d '\r' > ~/.ssh/id_rsa
+    - chmod 600 ~/.ssh/id_rsa
+    - rsync -avrz --delete -e  "ssh -p 22 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null" ${artifacts_dir}/* inbound@dispatch.pyr8.io:/home/inbound/cd/artifacts-java10/${CI_COMMIT_REF_NAME}/
```

Comment:
we should deduplicate that into an `.sh` script and use it here and above. Similar with other duplication in tasks above. We aren't dropping java8 support anytime soon, it seems to me.

Reply:
Makes sense.  I though we were going to ditch 8.